requirements/prod.txt

```diff
@@ -218,3 +215,8 @@ hashin==0.6.0 \
     --hash=sha256:7563490ec8c9c361e48c7623898b3585143d379714a03bee4c3d52fb13e85086
 django-import-export==0.4.5 \
     --hash=sha256:0406f781a771c39e325474df09644a5cea5188dfac7ecace4f32d664920abd00
+https://github.com/mozilla/mozilla-django-oidc/archive/0.1.2.zip#egg=mozilla_django_oidc \
```

Comment:
This means you also need to upgrade the rest of the ``mozilla-django-oidc`` dependencies.